ee/vellum_ee/workflows/display/utils/expressions.py

```diff
@@ -408,7 +408,24 @@ def serialize_value(executable_id: UUID, display_context: "WorkflowDisplayContex
                 },
             }
         else:
-            return {"type": "DICTIONARY_REFERENCE", "entries": cast(JsonArray, serialized_entries)}
+            # Check if this dictionary has a "role" field with USER or ASSISTANT, indicating it's a ChatMessage
+            role_entry = next((entry for entry in serialized_entries if entry["key"] == "role"), None)
```

Comment:
[blocking] is there a way we could do this for _all_ pydantic models instead of targeting `ChatMessage` ones based on interface?